force-app/main/default/lwc/pdftronAttachmentPickerCombobox/pdftronAttachmentPickerCombobox.html

```diff
@@ -1,33 +1,66 @@
 <template>
-    <lightning-card>
-        <div class="slds-var-p-around_xx-small slds-grid">
-            <template if:false={loadFinished}>
-                <div class="spinnerContainer">
-                    <lightning-spinner alternative-text="Loading" size="small"></lightning-spinner>
-                </div>
-            </template>
-            <div id="attachmentCombobox" class="slds-col slds-size_2-of-3">
-                <lightning-combobox
-                    class="slds-p-top_large slds-p-right_medium"
-                    name="progress"
-                    value={value}
-                    label="Choose an attachment"
-                    placeholder="Choose an attachment..."
-                    options={picklistOptions}
-                    variant="label-hidden"
-                    onchange={handleChange} ></lightning-combobox>
+    <div class="slds-grid slds-grid--vertical-stretch slds-p-bottom_x-small">
+        <template if:false={loadFinished}>
+            <div class="spinnerContainer">
+                <lightning-spinner alternative-text="Loading" size="small"></lightning-spinner>
             </div>
-            <div id="fileUploader" class="slds-p-around_xxx-large slds-col slds-size_1-of-3">
-                <lightning-file-upload
-                class="slds-p-top_xxx-small"
-                label="Attach document"
-                name="fileUploader"
-                accept={acceptedFormats}
-                record-id={recordId}
-                onuploadfinished={handleUploadFinished}
-                multiple>
-        </lightning-file-upload>
+        </template>
+        <div id="attachmentCombobox" class="slds-col slds-size_11-of-12">
+            <c-lookup onsearch={handleSearch} onselectionchange={handleSingleSelectionChange}
+                placeholder="Open File in WebViewer...">
+            </c-lookup>
+        </div>
+        <div id="fileUploader" class="slds-col slds-size_1-of-12 slds-p-left_xx-small">
+            <!-- lightning button for open modal window -->
+            <div>
+                <lightning-button-group>
+                    <lightning-button-icon icon-name="utility:add" onclick={openModal} variant="border-filled">
+                    </lightning-button-icon>
+                    <!--lightning-button-icon icon-name="utility:download" onclick={handleDownload}
+                        variant="border-filled"></lightning-button-icon>
+                    <lightning-button-icon icon-name="utility:search" onclick={handleClose}
+                        variant="border-filled"></lightning-button-icon-->
```

Comment:
Does this still need to be here if it's commented out? If so, perhaps a `@todo` is in order?

Reply:
Addressed in https://github.com/PDFTron/salesforce-webviewer-attachments/pull/7/commits/304bebb5a1f4c6e4457aa59eba1e279fa468a265